modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2208,9 +2208,12 @@ def apply_full_axis(
             The data types of the result. This is an optimization
             because there are functions that always result in a particular data
             type, and allows us to avoid (re)computing it.
-        keep_partitioning : boolean, default: True
-            The flag to keep partition boundaries for Modin Frame.
-            Setting it to True disables shuffling data from one partition to another.
+        num_splits : {"keep", "auto", int}, default: "keep"
```

Comment:
could have used an enum here but not sure whether it's convenient to mix enum values with actual integer values